SingularityUI/app/components/common/table/UITable.jsx

```diff
@@ -142,7 +144,7 @@ class UITable extends Component {
         rowData
       );
 
-      return <td key={tdIndex} className={col.props.className}>{cell}</td>;
+      return <td key={col.props.id} className={col.props.className}>{cell}</td>;
```

Comment:
are we worried about dupes here? do we have any way to enforce this ID being unique?